sourcegraph/cody
# Autocomplete: Mark Ollama support experimental

## Test plan

- Try with both `"cody.autocomplete.advanced.provider": "unstable-ollama"` and `"cody.autocomplete.advanced.provider": "experimental-ollama"`in the settings
- Check that it resolves to the correct provider by logging: 
    ```
    █ CodyCompletionProvider:initialized: experimental-ollama/deepseek-coder:6.7b-base-q4_K_M
    ```

<img width="808" alt="Screenshot 2024-02-07 at 20 21 37" src="https://github.com/sourcegraph/cody/assets/458591/ab7dbf4f-8e8e-4371-aef1-be9d7afc4b49">
